dashboard-frontend/src/pages/AccountsPage/InputAccountCredentials.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { type ChangeEventHandler } from 'react';
+import { classNames } from '../../helpers/strings';
+import startCase from 'lodash/startCase';
+
+type inputAccountCredentials = {
+  index: number;
+  error: string | undefined;
+  handleCredentialChange: ChangeEventHandler;
+  entity: string;
+  credentials: any;
+};
+
+const InputAccountCredentials: React.FC<inputAccountCredentials> = ({
+  error,
+  handleCredentialChange,
+  index,
+  entity,
+  credentials,
+}) => (
+  <React.Fragment>
+    <NameInput name={entity} index={index} />
+    <CredentialsInputList
```

Comment:
Yep, I like it!

Reply:
The only thing is readability, I'd move as many of those React F.Cs as possible into their own files and import them in.

So `NameInput`, `CredentialsInputList`, and `CredentialsInput` can all be abstracted away. As it will increase the number of files within the `AccountsPage` folder it might also be worth creating a `components` folder where you can move stuff like this. 

For study confs for example I have a structure like this `components/inputs`

Reply:
That's just my personal preference though, do what makes sense to you!

Reply:
On second thoughts, I'd probably still keep `CredentialsInputList` and `CredentialsInput` together as they're closely tied and it's pretty easy to see that one is just a list version of the other